src/components/Logo/Logo.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './styles.scss';
+
+export const Logo = () => {
+    return (
+        //TODO: change srcSet urls to 'src/assets/logos' image assets
+        <picture>
+            <source media="(min-width: 768px)" srcSet='https://via.placeholder.com/60x60.png?text=60x60+Logo'/>
```

Comment:
For the small logo, your `media` should consider the height not the width as the navbar (where the small one is going) will still share the same width as the page

Reply:
Oh I thought the Logo in this case was not in the Navbar, but in the Header above Navbar. What dimensions are appropriate in that case for height? I used 3 dimensions sizes for mobile, tablet & desktop so far but I'm not sure what's appropriate.

Reply:
I was thinking the Logo component would be reused in the Landing page and in the navbar, which should ultimately resize

We won't need different image source for different device (Except for desktop where we'd need something bigger)